src/sdl2_wrapper.cpp

```diff
@@ -717,17 +717,37 @@ CAMLprim value Val_SDL_Event(SDL_Event *event) {
   case SDL_PANEVENT:
     v = caml_alloc(1, 24);
 
-    vInner = caml_alloc(9, 0);
+    vInner = caml_alloc(5 * 8, 0);
```

Comment:
5 * 8 ???? why.

Reply:
me being dumb, was adjusting number of fields and forgot to change to constant

thanks for the catch